docsite/rst/YAMLSyntax.rst

```diff
@@ -42,6 +42,24 @@ A dictionary is represented in a simple ``key: value`` form (the colon must be f
         job: Developer
         skill: Elite
 
+More complicated data structures are possible, such as lists of dictionaries, or dictionaries whose values are lists.  Or a mix of both::
```

Comment:
Editing suggestion for this sentence:  More complicated data structures are possible, such as lists of dictionaries, dictionaries whose values are lists, or a mix of both::